js/index.js

```diff
@@ -0,0 +1,102 @@
+function shuffle(array) {
+    let currentIndex = array.length,  randomIndex;
+
+    while (currentIndex !== 0) {
+
+        randomIndex = Math.floor(Math.random() * currentIndex);
+        currentIndex--;
+
+        [array[currentIndex], array[randomIndex]] = [
+            array[randomIndex], array[currentIndex]];
+    }
+
+    return array;
+}
+
+let expectedWord = []
+fetch('words.json')
+    .then(response => response.json())
+    .then(words => {
```

Comment:
Currently, `fetch` returns an error (see below). 

Tried to see whether uploading the `words.json` file onto cloudinary helps, and it did! 
Can you guys upload `words.json` and replace the file path with the cloudinary url?

Error message: 
`Access to fetch at 'file:///Users/henrylee/sites/academyServer/html/2023-mar-wordle/words.json' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, isolated-app, chrome-extension, chrome, https, chrome-untrusted.`

Reply:
Resolved - had to be using localhost